reckruitment/index.php

```diff
@@ -35,47 +37,66 @@
+    <li><strong>Tuesday, January 14th</strong>: Skiles 309 from 7:00pm-8:00pm</li>
+    <li><strong>Thursday, January 16th</strong>: Skiles 309 from 7:00pm-8:00pm</li>
+    <li><strong>Tuesday, January 21st</strong>: Skiles 309 from 7:00pm-8:00pm</li>
 <!--    <li><strong>First Social: Monday, January 28th at 7:15pm (Mandatory)</strong></li>-->
-  <li>Check back here in December 2019 for info session dates for the 2020 recruitment cycle!</li>
+<!--  <li>Check back here in December 2019 for info session dates for the 2020 recruitment cycle!</li>-->
 </ul>
-
+<br>
+<h4 class="mb-3" id="first-social">First Social (Mandatory)</h4>
     <p>Attending this social is your chance as an applicant to meet Reck Clubbers for the first time, get a feel for what Ramblin’ Reck Club is all about, ask any questions you may have about the Club or getting involved, and most importantly, sign up for an interview time. No need to stress about this event; be yourself and talk to anyone you do not already know! Dress code for this event is <strong>business casual</strong>, and will be hosted in the Edge Building Trophy Room (on the corner of Techwood and Bobby Dodd).</p>
+    <ul>
+        <li><strong>First Social: Monday, February 3rd</strong></li>
+    </ul>
+    <p>There are two time slots for the first social: <strong>7:00pm-8:00pm and 8:30pm-9:30pm</strong>. Please note down your availability on the application. If you are not available during either of these times, please email <a href="mailto:membershipchair@reckclub.org">Kassie Lee</a>.</p>
 
-<h4 class="mb-3">RRC/Athletics Bonding (Mandatory)</h4>
+<br>
+<h4 class="mb-3" id="gtky-events">RRC “Get To Know You” Events (Mandatory)</h4>
 
-<p>In order to provide applicants a casual setting to interact with Reck Clubbers and get a better feel for the kind of spirit Reck Club has for Georgia Tech, all applicants must attend one of the following GT Athletic events. Please note that applicants need only attend one of the following athletic events in order to satisfy the recruitment requirements:</p>
+    <p>In order to provide applicants a casual setting to interact with Reck Clubbers and get a better feel for the kind of spirit Reck Club has for Georgia Tech, all applicants must attend one of the following GT Athletic events. Please note that applicants need only attend one of the following athletic events in order to satisfy the recruitment requirements:</p>
 
 <ul>
-<!--    <li><strong>Sunday, January 20th</strong>: Women’s Basketball game vs. Syracuse at 2pm in McCamish Pavilion</li>-->
-<!--    <li><strong>Wednesday, January 23rd</strong>: Tech Rec Bowling at 7pm in the Student Center</li>-->
-<!--    <li><strong>Sunday, January 27th</strong>: Women’s Basketball game vs. NC State at 2pm in McCamish Pavilion</li>-->
+    <li><strong>Thursday, January 23rd</strong>: Women’s Basketball game vs. UNC at 6pm in McCamish Pavilion</li>
```

Comment:
Move the event times next to the dates and bold them and the location; ex:

**Thursday, January 23rd (6pm)**: Women's Basketball vs UNC at **McCamish Pavilion**

Reply:
Alternatively:

**Thursday, January 23rd (6pm, McCamish Pavilion)**: Women's Basketball vs UNC